examples/live-list-update-reverse.amp.html

```diff
@@ -0,0 +1,167 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Lorem Ipsum | PublisherName</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Georgia|Open+Sans|Roboto' rel='stylesheet' type='text/css'>
+  <style amp-custom>
+    amp-live-list > [items] {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .amp-live-list-item {
+      height: 100px;
+      background-color: #f8f8f8;
+      border-bottom: 1px solid rgba(0, 0, 0, .12);
+    }
+
+    .card {
+      display: flex;
+      margin: 15px;
+      padding: 10px;
+      justify-content: center;
+      align-items: center;
+      background-color: #fff;
+      box-shadow: rgba(0, 0, 0, .3) 0 0 5px;
+    }
+
+    .card > div {
+      flex-basis: 50%;
+    }
+
+    .side {
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+    }
+
+    .content {
+      font-size: 1.5rem;
+    }
+
+    header {
+      font-size: 2rem;
+    }
+
+    #live-list-update-button {
+      position: fixed;
+      top: 10px;
+      left: 50%;
+      transform: translateX(-50%);
+    }
+
+    .button {
+      border: none;
+      border-radius: 2px;
+
+      color: #fafafa;
+      height: 26px;
+      min-width: 32px;
+      padding: 0 16px;
+      margin: 0 16px;
+      text-transform: uppercase;
+      letter-spacing: 0;
+      cursor: pointer;
+      vertical-align: middle;
+      line-height: 26px;
+      text-align: center;
+      background: #3f51b5;
+    }
+
+    .pagination {
+      display: inline-block;
+      padding: 0;
+    }
+
+    amp-live-list [pagination] nav {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    .pagination li {
+      display: inline;
+      border: 1px solid black;
+      padding: 10px;
+      list-style-type: none;
+    }
+  </style>
+  <link href='https://fonts.googleapis.com/css?family=Georgia|Open+Sans|Roboto' rel='stylesheet' type='text/css'>
+  <!-- NOTE: This script tag makes this document not a valid AMP but is needed to turn on the experiment feature -->
+  <script>
+      (self.AMP=self.AMP||[]).push(function(AMP){
```

Comment:
without refreshing the page this won't do much on the first try. I would say just make a "Turn sort experiment on" button.

Reply:
@aghassemi can you explain why this wouldn't work on first load? I double checked and it seems to work. This script tag is non async and will be the first executed callback in the AMP pipeline so before amp-live-list even get initialized this experiment should have been set on the cookie already. LMK if i'm misunderstanding something or checked it wrong.

Reply:
nvm me , didn't realize it is `sync`